firefox_media_tests/playback/test_playback_limiting_bandwidth.py

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette import BrowserMobProxyTestCaseMixin
+from test_video_playback_base import TestVideoPlaybackBase
+
+class TestPlaybackLimitingBandwidth(TestVideoPlaybackBase, BrowserMobProxyTestCaseMixin):
+
+    def __init__(self, *args, **kwargs):
+        TestVideoPlaybackBase.__init__(self, *args, **kwargs)
+        BrowserMobProxyTestCaseMixin.__init__(self, *args, **kwargs)
+
+    def test_playback_limiting_bandwidth(self):
+        self.run_playback(timeout=30)
+        BrowserMobProxyTestCaseMixin.setUp(self)
```

Comment:
I don't know anything about BrowserMobProxy, but shouldn't its setUp() and tearDown() methods be called in TestPlaybackLimitingBandwidth.setUp() and TestPlaybackLimitingBandwidth.tearDown()? That is, you want these to be called before and after every network bandwidth test in this class.